src/js/components/CheckboxTable.js

```diff
@@ -75,21 +79,22 @@ class CheckboxTable extends React.Component {
   renderHeadingCheckbox() {
     let checked = false;
     let indeterminate = false;
-    let {allowMultipleSelect, checkedItemsMap, data} = this.props;
+    let {allowMultipleSelect, checkedItemsMap, disabledItemsMap, data} = this.props;
```

Comment:
More than 80 characters here. We have a rule to keep code (not strings and regexes) within an 80 character limit.